src/subdomains/supporting/payment/services/transaction.service.ts

```diff
@@ -120,6 +120,20 @@ export class TransactionService {
     });
   }
 
+  async getManualRefVolume(ref: string): Promise<{ volume: number; credit: number }> {
+    const { volume, credit } = await this.repo
+      .createQueryBuilder('transaction')
+      .select('SUM(refReward.amountInEur / user.refFeePercent)', 'volume')
+      .addSelect('SUM(refReward.amountInEur)', 'credit')
+      .leftJoinAndSelect('transaction.user', 'user')
+      .leftJoinAndSelect('transaction.refReward', 'refReward')
```

Comment:
Wouldn't `leftJoin` be enough without select? We only want to select volume and credit.